### PR TITLE
[MODINVSTOR-1102]-Added DCB Instance and holdings insert script

### PR DESCRIPTION
**Pull request by SinghAdes**
[MODINVSTOR-1102](https://issues.folio.org/browse/MODINVSTOR-1102)

### CODE REVIEW

**Review by DmytroBykov1 — APPROVED** [review]

**Review by gurleenkaurbp — APPROVED** [review]

File: src/test/java/org/folio/rest/InstallUpgradeIT.java [hunk 1]
```diff
@@ -164,8 +164,6 @@ public void upgradeFromLotus() {
       .put("parameters", new JsonArray()
         .add(new JsonObject().put("key", "loadReference").put("value", "true"))
         .add(new JsonObject().put("key", "loadSample").put("value", "true"))));
-
-    smokeTest();
```
[the remark this answers is not in the document]
Comment (SinghAdes) [reply src/test/java/org/folio/rest/InstallUpgradeIT.java:168]: fixed, please have a look. thanks

File: src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -1111,11 +1111,21 @@
...
     {
       "run": "after",
       "snippetPath": "addReindexJobResourceName.sql",
       "fromModuleVersion": "26.1.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "addDcbHolding.sql",
+      "fromModuleVersion": "26.1.0"
+    },
```
Comment (psmagin) [inline src/main/resources/templates/db_scripts/schema.json:1128]: `fromModuleVersion` have to be changed to the newest

**Review by psmagin — COMMENTED** [review]
Please update NEWS file

**Comment by psmagin** [comment]
Does these holdings_record and instance records are needed for all libraries? May there be a case if a library doesn't want to use DCB functionality?

**Comment by gurleenkaurbp** [comment]
> Does these holdings_record and instance records are needed for all libraries? May there be a case if a library doesn't want to use DCB functionality?


As of now there has not been a requirement to restrict the creation of these instance holdings as they will be created and be displayed on the UI for all the libraries with DCB title and has been discussed with the PO as well.

File: src/main/resources/templates/db_scripts/schema.json (hunk 2 above)
Comment (SinghAdes) [reply src/main/resources/templates/db_scripts/schema.json]: updated to 27.0.0

File: NEWS.md [hunk 3]
```diff
@@ -32,6 +32,7 @@
 
 ### Features
 * Added new column complete_updated_date into INSTANCE table that will be used in mod-oai-pmh module: [MODINVSTOR-1105](https://issues.folio.org/browse/MODINVSTOR-1105)
+* Added insert script for single instance and holdings , that will be used in mod-dcb : [MODINVSTOR-1102](https://issues.folio.org/browse/MODINVSTOR-1102)
```
Comment (psmagin) [inline NEWS.md:35]: This version has already released, please move this line to v27.1.0 release text block

File: src/main/resources/templates/db_scripts/schema.json [hunk 4]
```diff
@@ -1111,11 +1111,21 @@
...
     {
       "run": "after",
       "snippetPath": "addReindexJobResourceName.sql",
       "fromModuleVersion": "26.1.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "addDcbHolding.sql",
+      "fromModuleVersion": "27.0.0"
```
Comment (psmagin) [inline src/main/resources/templates/db_scripts/schema.json:1127]: "27.0.0" version already released please use version from pom.xml, it's 27.1.0

**Review by psmagin — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 3 above)
Comment (SinghAdes) [reply NEWS.md]: fixed

File: src/main/resources/templates/db_scripts/schema.json (hunk 4 above)
Comment (SinghAdes) [reply src/main/resources/templates/db_scripts/schema.json]: fixed

File: src/main/resources/templates/db_scripts/addDcbHolding.sql [hunk 5]
```diff
@@ -0,0 +1,18 @@
+INSERT INTO ${myuniversity}_${mymodule}.holdings_record(id, jsonb)
+  SELECT '10cd3a5a-d36f-4c7a-bc4f-e1ae3cf820c9',
+    jsonb_build_object(
+      'hrid', 'DCB',
+      'instanceId', '9d1b77e4-f02e-4b7f-b296-3f2042ddac54',
+      'sourceId', id,
+      'permanentLocationId', (
+        SELECT id FROM ${myuniversity}_${mymodule}.location
+        WHERE jsonb ->> 'name' = 'Main Library'
```
[the remark this answers is not in the document]
Comment (gurleenkaurbp) [reply src/main/resources/templates/db_scripts/addDcbHolding.sql:9]: @julianladisch , 

As of now there has not been a requirement to restrict the creation of these instance holdings as they will be created and be displayed on the UI for all the libraries with DCB title and has been discussed with the PO as well.

CC: @kalibek-epam , @SinghAdes , @Vignesh-kalyanasundaram , @steveellis

**Pull request closed without merging**